src/components/TimelineNav/TimelineNav.stories.tsx

```diff
@@ -153,4 +153,8 @@ export default {
 
 type Args = React.ComponentProps<typeof TimelineNavPanel>;
 
-export const Default: StoryObj<Args> = {};
+export const Default: StoryObj<Args> = {
+  parameters: {
+    chromatic: { viewports: [414, 1336] },
```

Comment:
quick nit, how'd you pick 1336?

Reply:
we have some magic numbers across a few tests: 414, 768, 834, and 1366. I'm not yet sure why those values are used, but I figured they mapped to some breakpoints in the CSS

Reply:
Ahhh for context I picked 414 for iphone xr / pixel 2 (411 actually for pixel 2) the lowest , 768 for iPad mini, and 1366 for chromebooks, which was apparently a large share of devices.

Forget where 834 comes from tho lol

somewhat relevant but outdated:
https://summitlearning2.slack.com/archives/C039WM43Z16/p1652289796778229?thread_ts=1652228885.564429&cid=C039WM43Z16

Reply:
I'll create a ticket to add these as named constants to our preview.tsx file ([storybook docs](https://storybook.js.org/docs/react/essentials/viewport#configuring-per-component-or-story), [chromatic docs](https://www.chromatic.com/docs/viewports))

to reference:

- define the viewports we want
- use those in the storybook UI
- reference the widths in chromatic tests `viewports` param